### DISCUSSION TITLE
how i give a translation

### DISCUSSION THREAD

**Discussion by ZolOnTheNet** [Mezryss/FVTT-Genesys#110] [Q&A] [answered]
Hello !
I want to know How i give you a french translation of you Genesys Game. I not a translator but i'm very interresting by Genesys role playing and for my version, i have made a french translation file in attach 
[fr.json](https://github.com/Mezryss/FVTT-Genesys/files/13663863/fr.json)

**Comment by Assembling-Kings** [comment, marked as answer]
You are welcomed to provide a translation by doing a pull request and including the appropriate file. In this case a YAML file with the strings translated similar to this one: https://github.com/Mezryss/FVTT-Genesys/blob/main/yaml/lang/en.yml
